### DISCUSSION TITLE
为什么配置了config/routes.ts和config/config.ts不起作用呢？

### DISCUSSION THREAD

**Discussion by tanweiweiweiweiwei** [umijs/umi#6026] [Q&A]
按照官网的步骤配置的
// config/routes.ts

export default [
    { exact: true, path: '/', component: 'index' },
];

// config/config.ts

import { defineConfig } from 'umi';
import routes from './routes';

export default defineConfig({
  routes: routes,
});

**Comment by sorrycc** [comment]
信息太少了，帮不了你。

**Comment by cloudyan** [comment]
export default [
{ exact: true, path: '/', component: 'index' },
];


这个 component 写的不对，起码写个 '@/pages/index'，核对下文件路径
